yarn.lock

```diff
@@ -0,0 +1,3719 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Usually I use yarn as well. These days it is common to switch back to npm though. So you could instead add the `package-lock.json`. @Morishiri @ekorolev What do you think?

Reply:
I just used what was there already :) It's up to the authors which one to use, but I prefer npm as well ;)